src/bandersnatch/main.py

```diff
@@ -66,60 +69,82 @@ def mirror(config):
 
 
 def main():
-    parser = argparse.ArgumentParser(
-        description='PyPI PEP 381 mirroring client.')
-    parser.add_argument('-c', '--config', default='/etc/bandersnatch.conf',
-                        help='use configuration file (default: %(default)s)')
-    parser.add_argument('--debug', action='store_true', default=False,
-                        help='Turn on extra logging (DEBUG level)')
+    parser = argparse.ArgumentParser(description="PyPI PEP 381 mirroring client.")
+    parser.add_argument(
+        "-c",
+        "--config",
+        default="/etc/bandersnatch.conf",
+        help="use configuration file (default: %(default)s)",
+    )
+    parser.add_argument(
+        "--debug",
+        action="store_true",
+        default=False,
+        help="Turn on extra logging (DEBUG level)",
+    )
     subparsers = parser.add_subparsers()
 
     # `mirror` command
     m = subparsers.add_parser(
-        'mirror',
-        help='Performs a one-time synchronization with '
-             'the PyPI master server.')
-    m.set_defaults(op='mirror')
+        "mirror",
+        help="Performs a one-time synchronization with " "the PyPI master server.",
```

Comment:
This is one `black` thing I dislike. But get why hard for the AST to detect.
Please join the str

Reply:
fixed, and in the 4 other places this happened too